File: ProgrammingFundamentals/Java/04.MethodsDebuggingTroubleshootingCode/Lab/src/P07_MathPower.java
import java.text.DecimalFormat;
import java.util.Scanner;
public class P07_MathPower {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("#.#######");
        double number = Double.parseDouble(console.nextLine());
        int power = Integer.parseInt(console.nextLine());
        System.out.println(df.format(Power(number, power)));
    }

    static double Power(double number, int power) {
        double result = 1;
        for(int i = 0; i < power; i++) {
            result *= number;
        }
        return result;
    }
}

File: ProgrammingFundamentals/C#/04.MethodsDebuggingAndTroubleshootingCode/Lab/P07_MathPower/P07_MathPower.cs
using System;

namespace P07_MathPower
{
    class P07_MathPower
    {
        static void Main(string[] args)
        {
            double number = double.Parse(Console.ReadLine());
            int power = int.Parse(Console.ReadLine());
            Console.WriteLine(Power(number, power));
        }

        static double Power(double number, int power)
        {
            double result = 1;
            for (int i = 0; i < power; i++)
            {
                result *= number;
            }
            return result;
        }
    }
}
